api/features/serializers.py

```diff
@@ -244,6 +256,87 @@ class Meta:
     identity = _IdentitySerializer()
 
 
+class FeatureStateValueEdgeIdentityField(serializers.Field):
+    def to_representation(self, obj):
+        identity_id = self.parent._get_identity_uuid()
+        return obj.get_value(identity_id=identity_id)
+
+    def get_attribute(self, instance):
+        return instance
+
+    def to_internal_value(self, data):
+        feature_state_value_dict = FeatureState().generate_feature_state_value_data(
+            data
+        )
+
+        data = {**feature_state_value_dict}
+        fs_value_serializer = FeatureStateValueSerializer(data=data)
+        fs_value_serializer.is_valid(raise_exception=True)
+        return FeatureStateValue(**data).value
+
+
+class EdgeFeatureField(serializers.Field):
```

Comment:
Can we move all the fields into another `fields` module? Maybe that would be a bit ambiguous as to whether they were django model fields or serializer fields but I don't _really_ want to have to do like `serializer.fields`. What do you think?

Reply:
You mean just the serializer part, right? Or a new app with everything

Reply:
I was thinking the latter. Happy to discuss though.